gitpod-io/website
# :sparkles: Revamp `/cde` page

## Description

Revamp `/CDE` page

## Related Issue(s)



Fixes MKWEB-25

## How to test


- Open preview
- Go to `/cde`
- See 👀 



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3721"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
